### ISSUE TITLE
Loading issue

### ISSUE THREAD

**Issue by slayerbeast360** [LAX1DUDE/eaglercraft#58]
So whenever I try and load it up it shows this weird rainbow
![Screenshot 2022-02-01 12 24 34 PM](https://user-images.githubusercontent.com/98838954/152019214-8bb95cac-f7aa-4209-bb10-6fcc5a268e8d.png)

**Comment by sussygamedeveloper** [comment]
@slayerbeast360  are you on a arm chromebook?

**Comment by slayerbeast360** [comment]
> @slayerbeast360 are you on a arm chromebook?

No, I'm on a Lenovo 100e Chromebook Gen 2 (11.6"), running with Integrated Intel® UHD Graphics 600

**Comment by sussygamedeveloper** [comment]
maybe try updating chrome os? is it school issued? maybe its a hardware issue?

**Comment by slayerbeast360** [comment]
> maybe try updating chrome os? is it school issued? maybe its a hardware issue? also noticed that the skin is fucked up, can u send a screenshot of in-game?
![Screenshot 2022-02-01 1 10 10 PM](https://user-images.githubusercontent.com/98838954/152026141-21a9fc5b-b2fc-4f8e-9c6b-46573b9f18ea.png)

**Comment by sussygamedeveloper** [comment]
looks kinda like the arm graphics issue.

**Comment by LAX1DUDE** [comment]
this is a driver issue, the developers who maintain ANGLE don’t have time to perform extensive unit tests on sandboxed embedded SOC devices there’s not enough tools to correctly debug the drivers especially on something like a chromebook running digitally signed chromeos

This specific glitch is a clone of Issue [https://github.com/LAX1DUDE/eaglercraft/issues/11](https://github.com/LAX1DUDE/eaglercraft/issues/11) but I’ll leave it open because the people reporting issues here don’t know how to scroll

**Comment by slayerbeast360** [comment]
> this is a driver issue, the developers who maintain ANGLE don’t have time to perform extensive unit tests on sandboxed embedded SOC devices there’s not enough tools to correctly debug the drivers especially on something like a chromebook running digitally signed chromeos
> 
> This specific glitch is a clone of Issue #11 but I’ll leave it open because the people reporting issues here don’t know how to scroll

That makes sense, well thanks for your time, and ya they really don't.
